src/main/java/blackjack/domain/participant/rule/PlayerRule.java

```diff
@@ -0,0 +1,20 @@
+package blackjack.domain.participant.rule;
+
+import blackjack.domain.card.Deck;
+import blackjack.domain.participant.Player;
+import blackjack.view.InputView;
+import blackjack.view.OutputView;
+
+public class PlayerRule implements DrawRule<Player> {
+
+    private final InputView inputView = new InputView();
+    private final OutputView outputView = new OutputView();
+
+    @Override
+    public void draw(Player participant, Deck deck) {
+        while (participant.isDrawable() && inputView.isPlayerWantDraw(participant.getName())) {
```

Comment:
```suggestion
        while (participant.isDrawable() && inputView.isPlayerWantsToDraw(participant.getName())) {
```
개인적인 의견이지만, 메서드 이름은 문장처럼 읽히면 좋다고 생각해요

Reply:
저 역시 크게 공감합니다...! 하지만 영어가 익숙하지 않아...ㅎㅎ;;